tests/src/Kernel/Access/OgGroupContentOperationAccessTest.php

```diff
@@ -174,7 +174,7 @@ protected function setUp() {
     }
 
     // Create a 'blocked' user. This user is identical to the normal
-    // 'authenticated' member, except that she has the 'blocked' state.
+    // 'authenticated' member, except that it has the 'blocked' state.
```

Comment:
```suggestion
    // 'authenticated' member, except that they have the 'blocked' state.
```

(also change below it => they)